docs/madr/decisions/036-internal-listeners.md

```diff
@@ -0,0 +1,15 @@
+# Internal listeners naming convention
+* status: accepted
+
+## Context and Problem statement
+At the moment we don't clearly distinguish between listeners used for service to service communication
+and internal listeners, like Prometheus listeners.
+
+## Decision Outcome
+
+We will add prefix `_` to internal listeners.
```

Comment:
what are the implications of this? are there any downsides? for example we have metrics based on `envoy_cluster_name` so changing cluster names would influence metrics/dashboards/queries - for the listeners I only found https://github.com/kumahq/kuma/blob/7061107e586ef6b7dcf1eba453a76974addafef2/app/kumactl/data/install/k8s/metrics/grafana/kuma-dataplane.json#L1064 . Are there other things that we should look for?

Reply:
I've checked and didn't found any downsides of this, also we use this in MeshMetric right now, and no one found any issues with it